### ISSUE TITLE
Show data dir in any output

### ISSUE THREAD

**Issue by cblp** [ff-notes/ff#105]

**Comment by willbasky** [comment]
For example?

**Comment by cblp** [comment]
```
$ ff search 'data dir'
Actual:
  * Show data dir in any output
     
    | id B3Q7Q44H5ITT5-2003401L5E5UG
    | start 2018-11-27
    | tags [component_CLI, level_Beginner, type_Enhancement]
    | tracking https://github.com/ff-notes/ff/issues/105
No wikis to show
No contacts to show
database: /Users/cblp/dev/ff/.ff
```

**Comment by willbasky** [comment]
by option or every time?

**Comment by cblp** [comment]
let it be every time, unconditionally

**Comment by willbasky** [comment]
#200

**Comment by cblp** [comment]
fixed in #200
